packages/flutter_tools/lib/src/native_assets.dart

```diff
@@ -5,6 +5,7 @@
 // Logic for native assets shared between all host OSes.
 
 import 'package:logging/logging.dart' as logging;
+import 'package:native_assets_builder/native_assets_builder.dart';
```

Comment:
why are we importing the same library twice?

Reply:
We have conflicting types in the library and this file, so we need an import with a name for qualified access.
However, using all types from that library with a qualified name makes things very verbose.
Any suggestions?

Reply:
Ahh, I see. could you hide the ambiguous identifier from the unqualified import, to make it more clear what's going on?

Reply:
or I guess, hide the import from the other library and then only use the unqualified import?